### ISSUE TITLE
Views with multiple data export are hard to distinguish

### ISSUE THREAD

**Issue by steinmb** [elmcip/elmcip#431]
Discovered while working on issue #190. 

When you attach an export to a view page it is visualized by a icon. Having only one export and it is not a big issue though there might be accessibility issues but adding more then one and there is no way of knowing what is what. The only way to get information is by hovering on the link. Modern browsers have link url turn off by default and on mobile, tablets - this is almost impossible to pull off. There is also a accessibility issue lacking data export contextual information.

Here is a example mockup. Added link text but also grouped the RSS information. Actually is all four of them links and should be grouped.

## Current layout

<img width="438" alt="creative_works___elmcip" src="https://user-images.githubusercontent.com/683617/32772496-4e9b8b20-c926-11e7-9420-ac31f784cd89.png">

## Mockup example

![capto_capture 2017-11-14_10-35-20_am](https://user-images.githubusercontent.com/683617/32772879-a00f22b8-c927-11e7-86ab-4ac417f1b4c1.png)
